lib/reek/smells/uncommunicative_parameter_name.rb

```diff
@@ -60,10 +56,9 @@ def examine_context(ctx)
 
       private
 
-      def bad_name?(ctx, name)
-        sanitized_name = sanitize name
-        return false if sanitized_name == '*' || accept_names(ctx).include?(sanitized_name)
-        reject_patterns(ctx).any? { |pattern| sanitized_name.match pattern }
```

Comment:
I decided to NOT litter our code base with 

      # FIXME: switch to required kwargs when dropping Ruby 2.0 compatibility

We'll just have to grep for "raise" in lib once we drop support for 2.0.